_notices/rsn0011.md

```diff
@@ -0,0 +1,33 @@
+---
+layout: notice
+parent: RAPIDS Support Notices
+grand_parent: RAPIDS Notices
+nav_exclude: true
+notice_type: rsn
+# Update meta-data for notice
+notice_id: 11 # should match notice number
+notice_pin: true # set to true to pin to notice page
+title: "Minimum driver requirement for CUDA 11.0 in v21.10"
+notice_author: RAPIDS TPM
+notice_status: Completed
+notice_status_color: green
+# 'notice_status' and 'notice_status_color' combinations:
+#   "Proposal" - "blue"
+#   "Completed" - "green"
+#   "Review" - "purple"
+#   "In Progress" - "yellow"
+#   "Closed" - "red"
+notice_topic: Platform Support Change
+notice_rapids_version: "v21.10+"
+notice_created: 2021-10-12
+# 'notice_updated' should match 'notice_created' until an update is made
+notice_updated: 2021-10-12
+---
+
+## Overview
+
+We are increasing the minimum driver version for RAPIDS CUDA 11.0 to 450.80.02+ in order to support CUDA Enhanced Compatibility 
+
+## Impact
+
+Users would have to install CUDA driver 450.80.02+ in order to use CUDA 11.0
```

Comment:
I would say something like: Users should upgrade their driver to `450.80.02` or newer to continue using RAPIDS in the future